app.js

```diff
@@ -35,14 +35,23 @@ import parseResults from './results.js';
 import {getStorage} from './storage.js';
 import {parseUA} from './ua-parser.js';
 import Tests from './tests.js';
+import {exec} from './scripts.js';
 
 const storage = getStorage();
 
 /* c8 ignore start */
-const appVersion =
-  process.env.NODE_ENV === 'production' ?
-    (await fs.readJson(new URL('./package.json', import.meta.url))).version :
-    'Dev';
+let appVersion;
+if (process.env.NODE_ENV === 'production') {
+  appVersion = (await fs.readJson(new URL('./package.json', import.meta.url)))
+    .version;
+} else {
+  try {
+    appVersion = 'Dev-' + String(exec('git describe --tags')).substr(1);
```

Comment:
The `substr(1)` bit is assuming that that the tag starts with "v" and will result in something broken if that's not the case. How about we instead prepend a "v" to the version read from package.json, and make no assumptions about what the tag found by `git describe --tags` is?

See next comment for what to do about "Dev".